### ISSUE TITLE
您好，不知道您是否是原作者，我有一些细节想要咨询下。

### ISSUE THREAD

**Issue by Mark-sloan** [s7ev3n/DSNet#1]
搭建的模型没有用Imagemet数据集做预训练用CITYSCOPE数据集做的预训练就达到了良好的效果，是否有什么特殊技巧达到精度，平常工作中做很多从头训练的模型都不如用Imagemet数据集做预训练之后的效果。能不能麻烦留下联系邮箱，还有一些比较基础的问题想要询问您。

**Comment by s7ev3n** [comment]
Hi, @Mark-sloan 
Sorry for the delay.
As described in the paper, we did not pretrained on ImageNet, but on coarse labels of Cityscapes dataset. As in paper [Rethinking ImageNet Pre-training](https://arxiv.org/abs/1811.08883), pretraining on ImageNet "speeds up convergence early in training, but does not necessarily provide regularization or improve final target task accuracy."
For training tricks part, we believe large batch size(we use 16), multiple input sizes(we fit scale={0.5, 1.0}), and random flipping help to improve the final accuracy.

**Comment by Mark-sloan** [comment]
@s7ev3n Thank you very much for taking the time to answer for me. Hope to have a chance to communicate with you more.
